Description: Disable make check in the po folder as intltool-update gets
 confused by the files in quilt's .pc folder.
 Can be removed once http://bugs.debian.org/560704 is fixed.
Author: Felix Geyer <fgeyer@debian.org>

Index: purple-plugin-pack-2.6.2/po/Makefile.in.in
===================================================================
--- purple-plugin-pack-2.6.2.orig/po/Makefile.in.in	2010-02-03 19:39:52.000000000 +0100
+++ purple-plugin-pack-2.6.2/po/Makefile.in.in	2010-02-03 19:41:48.000000000 +0100
@@ -141,12 +141,8 @@
 	  rm -f $(DESTDIR)$(itlocaledir)/$$lang/LC_MESSAGES/$(GETTEXT_PACKAGE).mo.m; \
 	done
 
-check: all $(GETTEXT_PACKAGE).pot
-	rm -f missing notexist
-	srcdir=$(srcdir) $(INTLTOOL_UPDATE) -m
-	if [ -r missing -o -r notexist ]; then \
-	  exit 1; \
-	fi
+check:
+	# do nothing as it messes up with quilt's .pc folder.
 
 mostlyclean:
 	rm -f *.pox $(GETTEXT_PACKAGE).pot *.old.po cat-id-tbl.tmp
